From 670ba3f929e57f979eca9312949e3f3cd32a71fe Mon Sep 17 00:00:00 2001
From: WangJing <wangjing@mprc.pku.edu.cn>
Date: Tue, 25 Oct 2011 11:00:25 +0800
Subject: [PATCH 021/641] UniCore64: Delete legacy ld option -p

Fixed error:
ld: unrecognized option '-p'
-p is an option related with pipeline, which is not used at all

Signed-off-by: WangJing <wangjing@mprc.pku.edu.cn>
---
 arch/unicore64/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/unicore64/Makefile b/arch/unicore64/Makefile
index a47729f..b3e5b3f 100644
--- a/arch/unicore64/Makefile
+++ b/arch/unicore64/Makefile
@@ -19,7 +19,7 @@ endif
 # Provide the path to use for "make defconfig".
 KBUILD_DEFCONFIG	:= $(ARCH)_defconfig
 
-LDFLAGS_vmlinux		:= -p --no-undefined -X
+LDFLAGS_vmlinux		:= --no-undefined -X
 
 OBJCOPYFLAGS		:= -O binary -R .note -R .note.gnu.build-id -R .comment -S
 
-- 
1.7.9.5

